### DISCUSSION TITLE
Why are some hyperdrives unavailable even when I host them?

### DISCUSSION THREAD

**Discussion by rzelnik** [beakerbrowser/beaker#1553] [General]
I added all registered users and some dat pages to My Library for hosting/seeding. I also checked "Host This Drive" on all of them. Despite of this, even when I see them in my library lists, some hyperdrives are unavailable - Beaker just don't show their content. Why is that? Shuldn't they be available at least in my local library?

**Comment by pfrazee** [comment]
That does seem odd. There are some last-minute breaking changes being made to the wire protocol so perhaps that's involved? If this is continuing after the 1.0-beta launch, let us know.

**Comment by alizehctl** [comment]
I just filed an issue for this https://github.com/beakerbrowser/beaker/issues/1579. Probably best to contiinue this discussion there
